src/LineChart.ts

```diff
@@ -71,7 +71,7 @@ module n3Charts {
       };
 
       // Trigger the create event
-      eventMgr.trigger('create');
+      eventMgr.trigger('create', scope.options);
```

Comment:
Hmmm yeah, but then I'll need to store it somewhere to not parse it twice (the update occurs after and I don't want to wait until then to trigger the create event, obviously)